src/Chapter08/Table08.01.DefaultImplementedInterfaceMembers.cs

```diff
@@ -168,12 +172,13 @@ public Person(string firstName, string lastName)
         public string FirstName { get; }
         public string LastName { get; }
 
+        #if COMPILEERROR // EXCLUDE
         // private protected interface members
         // are not accessible in derived classes.
-       
-        // public int PersonTitle => 
-        //      GetName().ToUpper();
-    }
+        public int persontitle => 
+            getname().toupper();
```

Comment:
Why was the casing on this changed?
```suggestion
        public int PersonTitle => 
            GetName().ToUpper();
```